### PR TITLE
Fix for the disabled effect of Button component.

### PR DESCRIPTION
**Pull request by Vedant-Manjrekar**
## Changes proposed
- Added disabled effect for clear/reset button when no tags are selected.
- Made some css tweaks for the alignment of tags and it's numbers. 

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5681"></a>

### CODE REVIEW

**Comment by FrozenHearth** [comment]
Can you please add a screenshot?

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
There seems to be multiple changes/fixes in 1 PR, please only do the fix not other changes

**Comment by Vedant-Manjrekar** [comment]
> Can you please add a screenshot?

Before: 
Screenshot 2023-03-28 at 1 51 16 PM

After: 
Screenshot 2023-03-28 at 8 18 52 PM

**Comment by Vedant-Manjrekar** [comment]
> There seems to be multiple changes/fixes in 1 PR, please only do the fix not other changes

Fix is one only, others changes are an attempt to improve the alignment of the tag numbers. Anyways, if you don't want I will undo those changes.

**Comment by eddiejaoude** [comment]
Those extra changes have broken the design in other places. Yes best to remove them so the PR is not blocked by additional changes.

Some examples of the knock on effects...


Screenshot 2023-03-29 at 07 08 49
Screenshot 2023-03-29 at 07 08 13

File: components/Button.js [hunk 1]
```diff
@@ -2,18 +2,19 @@ import Link from "./Link";
 
 export default function Button({
   text,
-  primary = false,
   disable,
+  primary = false,
```
Comment (eddiejaoude) [inline components/Button.js:6]: is the move required?

File: pages/map.js [hunk 2]
```diff
@@ -122,8 +122,8 @@ export default function Map({ data }) {
             <Button
               onClick={resetFilter}
               text="Clear/Reset Filters"
-              primary={false}
               disable={selectedTags.size == 0 ? true : false}
+              primary={false}
```
Comment (eddiejaoude) [inline pages/map.js:126]: Not sure if this move is required?

Comment (Vedant-Manjrekar) [reply pages/map.js:126]: Yes, it was not required, it happened when I pulled changes from main branch.

Comment (eddiejaoude) [reply pages/map.js:126]: Probably best to revert these changes, to reduce conflicts for others

**Comment by Vedant-Manjrekar** [comment]
> Thank you for fixing 👍

Most welcome 😇

**Pull request merged**